ApollosProject/apollos-prototype
# Fixes Recently Liked

## DESCRIPTION

### What does this PR do, or why is it needed?

- The "Recently Liked" screen previously took a while to update when liking / unliking items. This PR fixes that. 
- The "Recently Liked" screen previously did not have a consistent order. This PR also fixes that and sorts items by most recently liked.

### What design trade-offs/decisions were made?

- I saw some performance improvements that could be made on the API side that I did not make at this time b/c I want to keep the size of this PR down.

### How do I test this PR?

1. Launch the app.
2. Unlike all items from your "Recently Liked" feed. They should disappear from the feed as soon as you unlike them (no need to refresh or wait).
3. Like a bunch of items on the home feed. They should appear in the "Recently Liked" feed at once, without needing to refresh. They should also be sorted with the most recently liked first.

### What automated tests did you write?

## TODO:

- [x] Closes #347 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
